archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -753,8 +763,24 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
+																<br/>
+																<ol start="2">
+																	<li><strong style="font-weight: bolder;">DevOps is a Cure-It-All Approach</strong></li>
+																</ol>
+																<div dir="ltr" style="text-align: justify;"><span style="font-size:16px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif">Here’s the harsh truth: Neither <a href="https://hackernoon.com/safe-a-scaled-agile-framework-in-a-nutshell-b4103170" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;">Agile</a> (Waterfall, Scrum - choose your option) nor DevOps is a panacea. DevOps as an ethos has the ability to uplift the current ailing software practices and is touted as <a href="https://hackernoon.com/an-introduction-to-agile-practices-why-small-release-in-scrum-fk2m34h9" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;">the key to quality software</a>. While this is certainly true, this philosophy can also pose potential risks and minor inconveniences like software feature bloating or narrow team member roles. So don’t expect it to take you from zero to a hero like a Fairy Godmother. Not to mention that it’s not an instant, off-the-shelf solution.</span></span></div>
+																<br/>
+																<div dir="ltr" style="text-align: justify;"><span style="font-size:16px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif"><p dir="ltr" style="text-align: center;line-height: 150%;margin: 10px 0;padding: 0;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;"><span style="font-family:georgia,times,times new roman,serif"><span style="font-size:17px"><a href="https://media.giphy.com/media/WsM7IXlbKWTzJQRLFg/giphy.gif" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;"><img data-file-id="1041068" src="https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2021/Feb/16.02/image2.gif" style="border: 0px initial;width: 220px;height: 134px;margin: 0px;outline: none;text-decoration: none;-ms-interpolation-mode: bicubic;" alt="visit hackernoon.com"></a></span></span></p></span></span></div>
+																<br/>
+																<ol start="3">
+																	<li><strong style="font-weight: bolder;">DevOps Uses Only Open-Source Software</strong></li>
```

Comment:
Same issue

Reply:
use this https://github.com/atherdon/newsletters/tree/master/archive/Thematic/snippets#h2